Brokerages/Bitfinex/BitfinexSymbolMapper.cs

```diff
@@ -91,7 +97,82 @@ public static List<Symbol> KnownSymbols
         {
             "BCHUSD","BCHBTC","BCHETH",
             "CFIUSD","CFIBTC","CFIETH",
-            "VENUSD","VENBTC","VENETH"
+            "VENUSD","VENBTC","VENETH",
+            "RRTBTC","QTMETH",
+            "AVTBTC","AVTETH",
+            "QSHBTC","QSHETH",
+            "YYWBTC","YYWETH",
+            "MNAETH","FUNBTC",
+            "SPKBTC","SPKETH",
+            "RCNBTC","RCNETH",
+            "RLCETH",
+            "AIDBTC","AIDETH",
+            "SNGBTC","SNGETH",
+            "ELFBTC","ELFETH",
+            "AIOETH",
+            "REQBTC","REQETH",
+            "RDNBTC","RDNETH",
+            "LRCETH","WAXETH",
+            "AGIBTC","AGIETH",
+            "BFTETH",
+            "MTNBTC","MTNETH",
+            "DTHBTC","DTHETH",
+            "MITBTC","MITETH",
+            "STJBTC","STJETH",
+            "XLMJPY",
+            "XVGEUR","XVGJPY","XVGGBP","XVGETH",
+            "BCIUSD","BCIBTC",
+            "KNCETH",
+            "POABTC","POAETH",
+            "LYMBTC","LYMETH",
+            "UTKBTC","UTKETH",
+            "VEEBTC","VEEETH",
+            "DADUSD","DADBTC","DADETH",
+            "ORSBTC","ORSETH",
+            "AUCBTC","AUCETH",
+            "POYBTC","POYETH",
+            "FSNBTC","FSNETH",
+            "CBTBTC","CBTETH",
+            "ZCNBTC","ZCNETH",
+            "SENUSD","SENBTC","SENETH",
+            "NCABTC","NCAETH",
+            "CNDBTC",
```

Comment:
Wonder whats the format logic behind having 1,2,3... symbols per line? Worth a comment I think

Reply:
One coin can have one or more tradable pairs;
when updating this list I just found it less error-prone to use one line per coin.

Reply:
Sounds good 👍. Couple of lines don't seem to respect it though: `"RRTBTC","QTMETH", "MNAETH","FUNBTC", "LRCETH","WAXETH",`